README.rst

```diff
@@ -17,6 +17,15 @@ You can install wellcome_aws_utils from PyPI::
 Note that this package **only supports Python 3.6 or later.**
 It was originally written to run inside the AWS Lambda environment, which supports 2.7 or 3.6 --- and we make heavy use of f-strings, which are 3.6 only.
 
+Running Tests
+*************
+
+First, run `export ROOT=$path to repo root$`
+
+To run tests, run `make test`
+
+Additional requirements should be added to `setup.py`. Adding requirements may require removal of the local `.tox` directory - run `rm -rf .tox`
```

Comment:
I’m pretty sure tox has a way to tell it to rebuild the venv, but I can’t remember the options right now.